langchain/chains/base.py

```diff
@@ -108,6 +108,8 @@ def __call__(
         inputs: Union[Dict[str, Any], Any],
         return_only_outputs: bool = False,
         callbacks: Callbacks = None,
+        *,
+        include_run_info: bool = False,
```

Comment:
Why make this optional?

Reply:
Mainly for better bw-compat, we have a bunch of existing tests that check the returned dict from `__call__` and expect it not to have a `__run` field. Other people might have tests like that too.